pkg/core/config/genesis/stage.json

```diff
@@ -1,6 +1,6 @@
 {
   "genesis_time": "2024-08-09T00:00:00.0000Z",
-  "chain_id": "audius-testnet-17",
+  "chain_id": "audius-testnet-18",
```

Comment:
this is fine to do but I don't understand why it's necessary for this change?

Reply:
Previously we did not include an SlaNodeReport in the actual rollup tx for validators that proposed 0 blocks. This was because we didn't know who all the validators were (features like registry_bridge hadn't been fixed at the time).

Now that we "know" who all the validators are, we can add these 0-blocks-proposed reports to the rollup tx, and we validate the tx expecting them to be there.

Reply:
i see thanks!